docs/security/pod-sandboxing-on-aks.mdx

```diff
@@ -0,0 +1,318 @@
+
+- You can further explore the mounted filesystem by navigating to the `/host-system/` path via `ls /host-system/`
+
+Using `chroot`, we can get direct access to host system privileges: `chroot /host-system bash`
+
+You should now see that you have access to host system resources by running `crictl pods`.
+
+Once we are done here, let's move on to Kata pods. First, let's clean up this deployment. The simplest way would be via `kubectl delete -f scenarios/system-monitor/`.
+
+### Kata Demonstration
+
+Now we want to see if running the pod as a Kata pod brings about any differences.
+
+Let's head over to the deployment YAML, which should be located at `...\kubernetes-goat\scenarios\system-monitor`.
+
+Open up the YAML in an editor of your choice.
+
+In the runtime spec, all you will need to do is add in one line:
+
+```YAML
+spec:
+  selector:
+    matchLabels:
+      app: system-monitor
+  template:
+    metadata:
+      labels:
+        app: system-monitor
+    spec:
+      runtimeClassName: kata-mshv-vm-isolation # one line change to make this pod a Kata sandbox.
+```
+
+Let's deploy the updated pod, and run the same tests as before:
+
+- We'll first exec into our pod (`kubectl exec -it <pod-name> -- bash`)
+- Run `capsh --print`. This time around, we should notice much less capabilities show up.
+- Run `mount`. Like `capsh`, we should see considerable less mounted resources than before.
+- Let's attempt to get direct access to host system privileges again, and see what host resources we have access to.
+  - We'll first run `chroot /host-system bash`.
+  - Next, we'll run `crictl pods`. This time around, we should be greeted with an error stating that the file/directory does not exist. From the Kata pod, we won't have access to host resources.
+
+With Kata pods, we can see that we generally have less (or in some cases, no) access to resources that we previously had in a non-Kata pod!
+
+To clean up this first demonstration, run `kubectl delete -f scenarios/system-monitor/` again.
+
+## Exploring pod sandboxing under the hood
+
+As mentioned earlier, Pod Sandboxing runs each pods as a virtual machine on top of your node OS.
+
+Let's take a look under the hood to see how Kata pods are set up.
+
+For this section, let's deploy some sample pods. You can find sample folder for this section under the `docs/getting-started/assets/pod-sandboxing-on-aks` in [AKS labs Github](https://github.com/Azure-Samples/aks-labs/tree/main/docs/security/assets/pod-sandboxing-on-aks/). Please download the folder and place it in the directory you are running the lab on.

```

Comment:
We should add very explicit directions on how to download these files instead of leaving it to the user to figure out.

```suggestion
For this section, let's deploy some sample pods. You can find sample folder for this section under the `docs/security/assets/pod-sandboxing-on-aks` in [AKS labs Github](https://github.com/Azure-Samples/aks-labs/tree/main/docs/security/assets/pod-sandboxing-on-aks/). Please download the folder and place it in the directory you are running the lab on.
```